Description: Allow running test using criterion 0.3
Author: Jan Niehusmann <jan@debian.org>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: jsonwebtoken/Cargo.toml
===================================================================
--- jsonwebtoken.orig/Cargo.toml
+++ jsonwebtoken/Cargo.toml
@@ -61,7 +61,7 @@ version = "0.6"
 optional = true
 
 [dev-dependencies.criterion]
-version = "0.4"
+version = ">= 0.3, < 0.5"
 
 [dev-dependencies.time]
 version = "0.3"
